tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -245,3 +245,24 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
     )
   });
 }
+
+if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
+  describe('Imagescan tests', { tags: '@p1'}, () => {
+    qase(112,
+      it("Fleet-112: Test imagescan app without expected semver range does not break fleet controller", { tags: '@fleet-112' }, () => {;
+        const repoName = 'local-cluster-imagescan-112'
+        const repoUrl = 'https://github.com/0xavi0/fleet-examples/'
```

Comment:
If possible, ask Xavi to move fleet-examples/imagescans data to fleet-test-data github repository.